packages/api/src/controllers/room.test.ts

```diff
@@ -0,0 +1,62 @@
+import serverPromise, { TestServer } from "../test-server";
+import { TestClient, clearDatabase, setupUsers } from "../test-helpers";
+import { User } from "../schema/types";
+import { RoomServiceClient } from "livekit-server-sdk";
+jest.mock("livekit-server-sdk");
+const MockedRoomServiceClient =
+  RoomServiceClient as jest.Mock<RoomServiceClient>;
```

Comment:
Maybe these should be the first lines in this file, just in case anyone changes the impl to use the mocked stuff statically (or this is really magic and I don't know how it works)

Reply:
What do you mean by using the mocked stuff statically?

Reply:
e.g.: if we moved the livekit client creation to the "file root" or server creation level, not creating one on every request

That could run before the mocks are setup and get the wrong result. Not a huge priority since it works now though, it's fine as is